lib/index.js

```diff
@@ -3,9 +3,41 @@
 exports.register = (server, options, next) => {
 
   server.ext('onPostAuth', (request, reply) => {
-    request.payload.loaded = true;
+    request.plugins['hapi-rate-limit'] = { rate: null };
+
+    if (!(request.route.method === 'get' || request.route.method === 'post' || request.route.method === 'delete')) {
+      return reply.continue();
+    }
+
+    const rate = Object.assign({}, options.defaultRate);
+    const routeSettings = request.route.settings.plugins.rateLimit || null;
```

Comment:
the `|| null` is unnecessary since `rateLimit` would just be false-y to begin with